models/models.js

```diff
@@ -173,3 +173,19 @@ exports.updateReviewVotes = (review_id, votes) => {
     })
 };
 
+
+
+
+exports.removeCommentByID = (comment_id) => {
+
+  if(isNaN(comment_id)){
+    return Promise.reject({status: 400, msg: 'comment_id must be an integer'})
+  }
+  return checkCommentExists(comment_id).then(() => {
+
+    return db.query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *`,[comment_id])
+  }).then((comment)=> {
+    return comment.rows[0]
+  })
```

Comment:
This is an interesting solution, in this case there is no need to make a separate call to a checkCommentsExists function. You already have a returning * here which can be used to determine if that comment does exist or not, so using a separate query in this case is unnecessary. You also do not need to return anything within this .then as that cannot be used in a 204 status request.